scripts/python/tao_build.py

```diff
@@ -336,7 +336,7 @@ def _write(line, cmd="build"):
                 _action_env("BUILD_WITH_AARCH64", "1")
                 _action_env("DISC_TARGET_CPU_ARCH", args.target_cpu_arch or "arm64-v8a")
         else:
-            if args.platform_alibaba and args.blade_gemm:
+            if args.blade_gemm:
```

Comment:
So are we going to enable blade_gemm in external code?

Reply:
This change is unexpected introduced by merge, will revert